src/Common/tests/System.Net/Sockets/Performance/SocketTestClient.cs

```diff
@@ -238,9 +238,23 @@ public Task<long> RunTest()
         }
 
         protected abstract string ImplementationName();
-        
-        //TODO: Either find a different fast way to compare or extract this in Interop.
```

Comment:
Please implement Compare as memcmp for Windows if this isn't possible for Linux.

In my investigations for WebSocket I noticed that this can have huge impact over the results and so we need to keep comparisons as fast as possible.